### ISSUE TITLE
GetNodeData in AuRa chains

### ISSUE THREAD

**Issue by LukaszRozmej** [openethereum/openethereum#546]
For questions please use https://discord.io/openethereum, issues are for bugs and feature requests.

_Before filing a new issue, please **provide the following information**._

- **OpenEthereum version : v3.3.0-rc.8
- **Operating system**: Windows / MacOS / Linux
- **Installation**: homebrew / one-line installer / built from source
- **Fully synchronized**: yes
- **Network**: xdai / EWF / POA Core / sokol / kovan / volta
- **Restarted**: N/A

This was mentioned before around London, but removing GetNodeData compromises Nethermind ability to FastSync from OE nodes, which in some Networks are still dominant nodes. this was already discussed here: https://github.com/openethereum/openethereum/issues/466

But still removed here because of some issues with Geth: https://github.com/openethereum/openethereum/pull/519

Could you enable GetNodeData on Aura chains?

**Comment by varasev** [comment]
@LukaszRozmej Yes, they removed it starting from v3.3.0-rc.8. Could you clarify if it works fine with OpenEthereum v3.3.0-rc.7? If so, we will try to enable it for AuRa chains.

**Comment by varasev** [comment]
@KaamilW Now you can try v3.3.0-rc.11: https://github.com/openethereum/openethereum/releases/tag/v3.3.0-rc.11
